### PR TITLE
Scheduler: Weekdays frequency should be 'weekly' instead of 'daily'

### PR DESCRIPTION
**Pull request by nmerchant**
When selecting 'Weekdays' as the repeat option of the scheduler control, the recurrencePattern is as follows:

`FREQ=DAILY;BYDAY=MO,TU,WE,TH,FR;INTERVAL=1`

when it should be:

`FREQ=WEEKLY;BYDAY=MO,TU,WE,TH,FR;INTERVAL=1`

This can be verified against the recurrencePattern that gets set if you choose 'Weekly' as the repeat option, and manually select each week day, as these are functionally identical.

### CODE REVIEW

File: js/scheduler.js [hunk 1]
```diff
@@ -554,14 +554,10 @@
 				}
 
 				if (recur.FREQ === 'DAILY') {
-					if (recur.BYDAY === 'MO,TU,WE,TH,FR') {
```
Comment (interactivellama) [inline js/scheduler.js]: Please allow "old API" to work here. We can add a comment:

```
// Weekdays should be set farther along in this conditional when frequency is `WEEKLY`, but in order to maintain backwards compatibility with an older version of `setValue` the following few lines set the UI to weekdays when frequency is `DAILY` also.
```

**Comment by interactivellama** [comment]
Please note this could be a breaking change if you are expecting the old `FREQ=DAILY;BYDAY=MO,TU,WE,TH,FR;INTERVAL=1` when the Repeat: Weekdays is selected.

**Comment by interactivellama** [comment]
@dkilgore Does this change of output RRULE string cause any issue with your team?

**Comment by kevinparkerson** [comment]
All checks out! Thanks @nmerchant

**Pull request merged**